fribidi-types.h: Fatal error:  'config.h' file not found

### What did you do?
build libraqm on macOS
option:
$ brew install libtiff libjpeg webp little-cms2
$ brew install freetype harfbuzz fribidi
$ depends/install_raqm_cmake.sh

### What did you expect to happen?
Building on macOS

### What actually happened?

-- Build files have been written to: /Users/alan/fork_work/Pillow/depends/raqm-cmake-b517ba80/build
Scanning dependencies of target raqm
[ 50%] Building C object CMakeFiles/raqm.dir/src/raqm.c.o
In file included from /Users/alan/fork_work/Pillow/depends/raqm-cmake-b517ba80/src/raqm.c:31:
In file included from /usr/local/include/fribidi/fribidi.h:31:
In file included from /usr/local/include/fribidi/fribidi-unicode.h:32:
/usr/local/include/fribidi/fribidi-types.h:31:11: fatal error: 'config.h' file not found
\# include <config.h>
          ^~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/raqm.dir/src/raqm.c.o] Error 1
make[1]: *** [CMakeFiles/raqm.dir/all] Error 2
make: *** [all] Error 2
~/fork_work/Pillow/depends
/usr/local/include/fribidi/fribidi-types.h:31:11: fatal error: 'config.h' file not found

### What versions of Pillow and Python are you using?
python2.7.14
Pillow5.1.0

@anpr No, still reproducible for me:

```

-- Build files have been written to: /Users/alexclark/Developer/pillow/Pillow/depends/raqm-cmake-b517ba80/build
Scanning dependencies of target raqm
[ 50%] Building C object CMakeFiles/raqm.dir/src/raqm.c.o
In file included from /Users/alexclark/Developer/pillow/Pillow/depends/raqm-cmake-b517ba80/src/raqm.c:31:
In file included from /usr/local/include/fribidi/fribidi.h:31:
In file included from /usr/local/include/fribidi/fribidi-unicode.h:32:
/usr/local/include/fribidi/fribidi-types.h:31:11: fatal error: 'config.h' file not found
# include <config.h>
          ^~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/raqm.dir/src/raqm.c.o] Error 1
make[1]: *** [CMakeFiles/raqm.dir/all] Error 2
make: *** [all] Error 2
~/Developer/pillow/Pillow/depends

```

--- a/depends/install_raqm_cmake.sh
+++ b/depends/install_raqm_cmake.sh
@@ -2,7 +2,7 @@
 # install raqm
 
 
-archive=raqm-cmake-b517ba80
+archive=raqm-cmake-99300ff3
 
 ./download-and-extract.sh $archive https://raw.githubusercontent.com/python-pillow/pillow-depends/master/$archive.tar.gz
 

